packages/ui5-config/src/ui5config.ts

```diff
@@ -34,18 +39,58 @@ import { fioriToolsProxy, serveStatic } from './constants';
  */
 export class UI5Config {
     private document: YamlDocument;
+    private static validate: ValidateFunction<SomeJSONSchema>;
+
+    /**
+     * Validates the schema of the given yaml document.
+     *
+     * @returns true if the document is valid, false otherwise
+     */
+    async validateSchema(): Promise<boolean> {
+        if (!UI5Config.validate) {
+            const path = join(__dirname, '..', 'dist', 'schema', 'ui5.yaml.json');
+            const schema = JSON.parse(await readFile(path, 'utf8')) as SomeJSONSchema | null;
+            if (!schema) {
+                throw Error('Schema file not found. No validation possible.');
```

Comment:
```suggestion
                throw Error('The schema file was not found. Validation is not possible.');
```

Reply:
Done in 604b3e90f6b1a742735db764bc0f38e1cb28a5e4